includes/wf_crm_webform_postprocess.inc

```diff
@@ -2233,8 +2219,10 @@ class wf_crm_webform_postprocess extends wf_crm_webform_base {
    *   Leave blank - used internally to recurse through data
    * @internal param $depth int
    *   Leave blank - used internally to track recursion level
+   * @param $customOnly
+   *   TRUE if only custom contact reference needs to be filled.
    */
-  private function fillContactRefs($values = NULL, $depth = 0) {
+  private function fillContactRefs($values = NULL, $depth = 0, $customOnly = FALSE) {
```

Comment:
Adding a new param on the end makes this function awkward to use. Better to add it to the beginning.